src/main/java/Calculator/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+package Calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
+
+    public static int AddNumbers(String text) {
+        if(text == null) return 0;
+        if(text.isBlank()) return 0;
+        return getSum(getIntegers(getValues(text)));
+    }
+
+    private static int[] getIntegers(String[] values) {
+        int[] numbers = new int[values.length];
+        for (int i=0; i<values.length; i++){
+            numbers[i] = Integer.parseInt(values[i]); // 숫자가 아닌 문자의 경우 NumberFormatException 발생
+            if(isNegative(numbers[i]) ==true) throw new RuntimeException("Negative Number Included");
```

Comment:
```suggestion
            if(isNegative(numbers[i]) throw new RuntimeException("Negative Number Included");
```

로 간단하게 표현하는 것도 좋습니다 :)